modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -11,10 +11,10 @@ You are responsible for arranging access to {centos} base media repositories and
 
 ifeval::[{suma-content} == true]
 
-[IMPORTANT]
+[WARNING]
 ====
-{suse} does not provide support for {centos} operating systems.
-{productname} allows you to manage {centos} clients, but support is not provided.
+The ``spacewalk-utils-extras`` package is not supported by {suse}, including the ``spacewalk-common-channels`` tool.
+You will not be supported if your system becomes inoperable by using these tools.
```

Comment:
I copied this from the Ubuntu client tools as it's what we have there.

Question is: SUSE Manager release notes says this: https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/4.0/#_spacewalk_utils

So as you can see, 3 scripts (now part of `spacewalk-utils`) are now fully supported (meaning L3 supported), while the others are L1 supported.

However we always said at the doc they were not supported. Is it the case? @paususe?